src/Symfony/Component/HttpKernel/Client.php

```diff
@@ -48,26 +48,6 @@ public function __construct(HttpKernelInterface $kernel, array $server = array()
     }
 
     /**
-     * {@inheritdoc}
-     *
-     * @return Request|null A Request instance
-     */
-    public function getRequest()
```

Comment:
I see the point, in this case following approach would work as well:
```php
/**
 * Client simulates a browser and makes requests to a Kernel object.
 *
 * @author Fabien Potencier <fabien@symfony.com>
 *
 * @method Request|null getRequest() A Request instance
 * @method Response|null getResponse() A Response instance
 */
```
Elegant and no additional code needed, what is your opinion aboutit?